compiler/src/org.graalvm.compiler.java/src/org/graalvm/compiler/java/LambdaUtils.java

```diff
@@ -51,6 +51,8 @@
 public final class LambdaUtils {
     private static final Pattern LAMBDA_PATTERN = Pattern.compile("\\$\\$Lambda\\$\\d+[/\\.][^/]+;");
```

Comment:
Can't we use this one for all we need?

Reply:
No, explanation is in the comment above.